src/components/common/Utils.ts

```diff
@@ -0,0 +1,58 @@
+export type Neighborhood = 4 | 8;
```

Comment:
I think we should have a 'utils' folder outside of components. That's what I'm doing with the simulation runner